symphony-client/src/main/java/org/symphonyoss/symphony/clients/impl/UsersClientImpl.java

```diff
@@ -93,7 +91,7 @@ public UsersClientImpl(SymAuth symAuth, SymphonyClientConfig config, Client http
 
         apiClient.setBasePath(config.get(SymphonyClientConfigID.POD_URL));
 
-
+        getAllUsersTimeout = Long.valueOf(System.getProperty("SYMPHONY_GET_ALL_USERS_TIMEOUT", "5")).longValue();
```

Comment:
Using hardcoded JVM system properties for this kind of thing is rather brittle, and complicates the use of techniques such as dependency injection (i.e. Spring) for bootstrapping code.

A better alternative would be to add this new timeout configuration option to the [`SymphonyClientConfigID` enum](https://github.com/symphonyoss/symphony-java-client/blob/master/symphony-client/src/main/java/org/symphonyoss/client/SymphonyClientConfigID.java#L19), and then fix any fallout elsewhere (e.g. ensuring it also gets added to [`SymphonyClientConfigBuilder`](https://github.com/symphonyoss/symphony-java-client/blob/master/symphony-client/src/main/java/org/symphonyoss/client/SymphonyClientConfigBuilder.java).

Reply:
Somewhat old code.  Agree with @pmonks we need to add it into the SymphonyClientConfigID etc..   This makes it more consistent with SJC configuration management.